src/forklift/cli.py

```diff
@@ -133,46 +136,66 @@ def start_lift(file_path=None):
 
     lift.process_crates_for(pallets, core.update)
 
-    log.info('elapsed time: %s', seat.format_time(clock() - start_seconds))
+    elapsed_time = seat.format_time(clock() - start_seconds)
+    log.info('elapsed time: %s', elapsed_time)
 
     pallet_reports = lift.process_pallets(pallets)
+    report_object = lift.create_report_object(pallet_reports, elapsed_time)
 
-    _send_report_email(pallet_reports)
+    email = get_config_prop('sendEmails')
+    if email:
+        _send_report_email(report_object)
 
+    print('Finished in %s. General email notification: %s', elapsed_time, email)
```

Comment:
nah i dont' think so, it's for the console. elapsed time is log.info'd it on line 140.